repository-data/webfiles/src/main/resources/site/freemarker/common/static-ui/news-hub-page.ftl

```diff
@@ -0,0 +1,70 @@
+<#ftl output_format="HTML">
+<#include "../../include/imports.ftl">
+
+<#include "../macro/sectionNav.ftl">
+<#include "../macro/tagNav.ftl">
+<#include "../macro/hubBox.ftl">
+<#include "../macro/stickyGroupBlockHeader.ftl">
+
+<article class="article article--news-hub">
+    <div class="grid-wrapper grid-wrapper--full-width grid-wrapper--wide" aria-label="Document Header">
+        <div class="local-header article-header article-header--with-icon">
+            <div class="grid-wrapper">
+                <div class="article-header__inner">
+                    <div class="grid-row">
+                        <div class="column--one-third column--reset">
+                            <img src="<@hst.webfile path="images/icon-article.png" fullyQualified=true/>" alt="News article">
+                        </div>
+                        <div class="column--two-thirds column--reset">
+                            <h1 class="local-header__title" data-uipath="document.title">News</h1>
+                            <p class="article-header__subtitle">The latest news and events from NHS Digital.</p>  
+                        </div>
+                    </div>
+                </div>
+            </div>
+        </div>
+    </div>
+
+    <div class="grid-wrapper grid-wrapper--article">
+        <div class="grid-row">
+            <div class="column column--one-third page-block page-block--sidebar sticky sticky--top">
+                <#assign filterNavLinks = [{ "url": "#", "title": "NHS Digital (4)" }, { "url": "#", "title": "Programme (16)" }, { "url": "#", "title": "NHS Digital (32)" }, { "url": "#", "title": "System and services (22)" }, { "url": "#", "title": "2018 (25)" }, { "url": "#", "title": "2017 (150)" }, { "url": "#", "title": "2016 (123)" }, { "url": "#", "title": "2015 (85)" }]/>
```

Comment:
Do the numbers in brackets denote the number of say 2017 news articles?  Are they intentionally hardcoded or were they a placeholder to be replaced with dynamic value?

Reply:
@ngta1 Everything under `static-ui` is for getting rapid UI work done without any backend - it also gives us a glimpse of the UI patterns and components we're using without the need of any dynamic data.

Reply:
Ah, of course.  Overlooked the static-ui part.  Thanks for clarifying.